build/PackageDiffIgnore.xml

```diff
@@ -1890,6 +1900,33 @@
 			<Member fullName="Microsoft.UI.Xaml.IXamlServiceProvider Uno.UI.Helpers.MarkupHelper.CreateParserContext(System.Object target, Microsoft.UI.Xaml.Markup.ProvideValueTargetProperty property)" reason="Not in WinUI; refactored out" />
 			<Member fullName="Windows.UI.Xaml.IXamlServiceProvider Uno.UI.Helpers.MarkupHelper.CreateParserContext(System.Object target, Windows.UI.Xaml.Markup.ProvideValueTargetProperty property)" reason="Not in WinUI; refactored out" />
 
+			<!-- BEGIN templated-parent rework -->
+
+			<!-- DependencyObjectStore -->
+			<Member fullName="System.Void Microsoft.UI.Xaml.DependencyObjectStore..ctor(System.Object originalObject, Microsoft.UI.Xaml.DependencyProperty dataContextProperty, Microsoft.UI.Xaml.DependencyProperty templatedParentProperty)" reason="Not in UWP" />
```

Comment:
We probably need this for backcompat, even if we don't use templatedParentProperty parameter.

Reply:
Yes, we need this for back compat, the generators create this method.

```
System.MissingMethodException: Method not found: void Microsoft.UI.Xaml.DependencyObjectStore..ctor(object,Microsoft.UI.Xaml.DependencyProperty,Microsoft.UI.Xaml.DependencyProperty)
   at CommunityToolkit.WinUI.Lottie.LottieVisualSourceBase.SetValue(DependencyProperty dp, Object value) in /agent/_work/1/s/src/AddIns/Uno.UI.Lottie/Uno.UI.SourceGenerators/Uno.UI.SourceGenerators.DependencyObject.DependencyObjectGenerator/CommunityToolkit.WinUI.Lottie.LottieVisualSourceBase_4797a3156fd2d9d24aa48d3f36c230dd.cs:line 65
   at CommunityToolkit.WinUI.Lottie.LottieVisualSourceBase.set_UriSource(Uri value) in /agent/_work/1/s/src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.cs:line 55
```